src/internal/orders/sales-order-time-range.ts

```diff
@@ -1,26 +1,19 @@
 import Joi = require("joi");
 import {
-  SalesOrderPaging as SalesOrderPagingPOJO,
   SalesOrderStatus,
-  SalesOrderTimeRange as ISalesOrderTimeRange,
-  TimeRangePOJO
+  SalesOrderTimeRange as ISalesOrderTimeRange
```

Comment:
The type alias should be `SalesOrderTimeRangePOJO` to be consistent with the rest of the SDK.

Reply:
Actually, this import shouldn't exist at all.  The original `SalesOrderTimeRangePOJO` interface should not have been deleted